docker/justfile

```
@@ -34,17 +34,24 @@ test *args="": copy-assets
 
 
 # run dev server in docker container
-serve: copy-assets
+serve env="dev" *args="": copy-assets
```

```suggestion
serve env="dev" *options="": copy-assets
```
the nittiest of nits

---

`docker compose up [OPTIONS] [SERVICE...]`

https://docs.docker.com/engine/reference/commandline/compose_up/

---

Oh thank you. I was copying how we did it elsewhere and noticed the difference but wasn't sure of a better name. That makes sense though.